spec/unit/lib/temporal/client_spec.rb

```diff
@@ -895,6 +895,32 @@ class NamespacedWorkflow < Temporal::Workflow
     end
   end
 
+  describe '#get_workflow_history' do
+    it 'gets full history with pagination' do
+      completed_event = Fabricate(:workflow_completed_event, result: nil)
+      response_1 = Fabricate(:workflow_execution_history, events: [completed_event], next_page_token: 'a')
+      response_2 = Fabricate(:workflow_execution_history, events: [completed_event], next_page_token: nil)
+
+      allow(connection)
+        .to receive(:get_workflow_execution_history)
+        .and_return(response_1, response_2)
+
+      subject.get_workflow_history(namespace: namespace, workflow_id: workflow_id, run_id: run_id)
+
+      expect(connection)
+        .to have_received(:get_workflow_execution_history)
+        .with(namespace: namespace, workflow_id: workflow_id, run_id: run_id, next_page_token: nil)
+        .ordered
+
+      expect(connection)
+        .to have_received(:get_workflow_execution_history)
+        .with(namespace: namespace, workflow_id: workflow_id, run_id: run_id, next_page_token: 'a')
+        .ordered
+
+      expect(connection).to have_received(:get_workflow_execution_history).exactly(2).times
+    end
+  end
```

Comment:
This test is failing in CI, could you take a look?

Reply:
should be fixed!